.gitignore

```diff
@@ -0,0 +1,25 @@
+
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
```

Comment:
LGTM but I don't really know the purpose of this commit. What does gitignore do?

Reply:
Basically the .gitignore says these are files that should never be pushed to the remote branch. For Java, this means we don't want any of the files that end in .class, along with some other files and file extensions

Reply:
LGTM